code/game/machinery/cryopod.dm

```diff
@@ -290,6 +290,11 @@
 					continue
 				O.loc = src
 
+	for(var/obj/machinery/computer/cloning/cloner in clonecomputers)
+		for(var/datum/dna2/record/R in cloner.records)
+			if(R.ckey == occupant.ckey)
```

Comment:
If someone ghosts in cryo, they'll be out of their body before this, and `occupant.ckey` will be null. Comparing `mind`s might do the trick, though.